examples/model_compress/pruning/auto_pruners_torch.py

```diff
@@ -62,30 +62,6 @@ def get_data(dataset, data_dir, batch_size, test_batch_size):
             ])),
             batch_size=batch_size, shuffle=False, **kwargs)
         criterion = torch.nn.CrossEntropyLoss()
-    elif dataset == 'imagenet':
```

Comment:
why imagenet is removed?

Reply:
BTW, what kind of pruners is auto pruner?

Reply:
suggest to add one more example which combines auto tune and pruner, for example, tuning the sparsity.

Reply:
suggest to add more detailed description at the top of each of these example code files.

Reply:
Imagenet is not used in comparison experiment, remove it for simplicity and clarity. Add more description at the top of the file, please review the latest version.